admin/users.js

```diff
@@ -32,13 +32,24 @@ module.exports = function (client) {
         .get(`${baseUrl}${encodeURIComponent(id)}`)
         .then(res => res.body);
     },
-    list (params) {
+    list (params = {}) {
+      let results = [];
+
+      const fetchNext = (res) => {
```

Comment:
why do you need this page by page loader if you can request all users\apps at once 
by passing `count` parameter as something big like Number.MAX_SAFE_INTEGER 
https://github.com/ExpressGateway/express-gateway/blob/f42d8d1fe61c74779a70ede452255583fa6a430f/lib/services/consumers/application.dao.js#L56

Reply:
Using a large number in theory guarantees that I get all the users — in practice we can't never be completely sure. We do not know how large the data base is.

Also — if I recall correctly the count is about the number of fields, not the number of records.

I do not feel really comfortable in making a single call with a large number and cross the fingers that all will work accordingly. If it's a paginated resource, we should handle pagination.

@kevinswiber Thoughts?